pkg/generate/app/cmd/newapp.go

```diff
@@ -883,6 +884,15 @@ func (c *AppConfig) Run() (*AppResult, error) {
 			}
 		}
 	}
+	if c.SourceSecret != "" {
```

Comment:
nit: we usually check for non-empty strings with `len(c.SourceSecret) > 0`

Reply:
updated in next push